### ISSUE TITLE
Can't interact with interactive elements inside a figcaption element (JAWS/Chrome)

### ISSUE THREAD

**Issue by kyle-wilson** [FreedomScientific/standards-support#120]
### Summary

While using JAWS (keyboard navigation) with Chrome, the user cannot interact with (click/submit) interactive elements (buttons, links, etc.) that are inside a figcaption element (within a figure). The user is able to tab-focus these elements, but pressing enter (or spacebar) does the following:
1. Focuses the previously-focused element
2. Fires a click event at the previously-focused element.

Scenario 1:
1. Go to https://codepen.io/kwilsonIXL/full/QZWLbx/
2. Tab to button B via button A. Press enter or spacebar.

Scenario 2:
1. Go to https://codepen.io/kwilsonIXL/full/QZWLbx/
2. Tab to button C via buttons A and B.
3. Shift-Tab to button B. Press enter or spacebar.

### Expected result

It is expected that the counter adjacent to button B is incremented and focus remains on button B.

### Actual result

In both scenarios, the previously-focused element is given focus and the counter adjacent to that button is incremented. 

- In the case of Scenario 1, button A is focused and its counter is incremented. Button A retains focus after the interaction. 
- In the case of Scenario 2, button C is focused and its counter is incremented. Button C retains focus after the interaction.

### Additional Information

In Firefox, the same behavior is reproducible but occurs significantly less often. Using the scenarios above, the actual behavior is almost always the expected behavior: the click event is fired at button B and button B retains focus after the interaction. Rarely do we encounter the unexpected behavior.

### Example

https://codepen.io/kwilsonIXL/full/QZWLbx/

Another use case with the details and summary elements:
https://codepen.io/kwilsonIXL/full/vVYBPp/ 

#### JAWS version and build number
Version 2018.1808.10 ILM

#### Operating System and version
Windows 10 Pro Version 1803 (OS Build 17134.285)

#### Browser and version:
Chrome 69.0.3497.100
Firefox 62.0.2

**Comment by JAWS-test** [comment]
The entire content in figcaption is not available for JAWS, it is not output when reading with arrow keys, it is not focused when using quick navigation (B), and it is not displayed in the element list (Crtr+Ins+B).

**Comment by JAWS-test** [comment]
If the graphic receives an alternative text, the switch is output, but still cannot be operated.

**Comment by stevefaulkner** [comment]
Could reproduce in Chrome Version 75.0.3770.142 with JAWS 19, but not in Firefox 69.0b5 (64-bit)

Seems to be due to JAWS not liking the `figcaption` providing the accessible name for the `figure` in Chrome as when that relationship is broken by providing an accessible name for the `figure` using `aria-label` on the figure, the interaction/announcement of the figcaption content works fine; [test case](https://codepen.io/stevef/pen/ZgYaQg) This method could be used to overcome the issue in Chrome until it is fixed in JAWS (will file internal bug). The larger question is that the inclusion of interactive controls (and thus the surfacing of their accessible names as labels for the `figure` is sub-optimal) The `figcaption` is supposed to contain a description of the `figure` content.

**Comment by JAWS-test** [comment]
According to specification links and buttons in figcaption are allowed. And I think they also make sense: e.g. linking the detailed text alternative for blind people.

**Comment by mitchellevan** [comment]
> According to specification links and buttons in figcaption are allowed. And I think they also make sense: e.g. linking the detailed text alternative for blind people.

+1. This bug happened to me today in real world content, with an in-page link to a footnote. Example (simplified):

```html
<figure>
  <figcaption>A complex chart<a href="#fignotes"><sup>1</sup></a></figcaption>
  <img src="..." alt="...">
  ... image long description ...
  <p id="fignote1" tabindex="-1"><sup>1</sup>Source: ABC123</p>
</figure>
```

**Comment by stevefaulkner** [comment]
tested in JAWS 2023.2307.37 AND CHROME Version 115.0.5790.171 (Official Build) (64-bit) COULD NOT REPRODUCE.
